src/test/java/uk/gov/pay/adminusers/service/ExistingUserInviteCompleterTest.java

```diff
@@ -86,6 +86,29 @@ public void shouldSuccess_whenSubscribingAServiceToAnExistingUser_forValidInvite
         assertThat(persistedUser.getValue().getServicesRole(service.getExternalId()).isPresent(), is(true));
     }
 
+    @Test
+    public void shouldSuccess_whenSubscribingAServiceToAnExistingUser_forValidInvite_with2FAMethod() {
```

Comment:
In this case completing the invite does not create a user as it already exists and we're not doing anything with the second factor method passed in, so I don't think this is testing anything. I think it's just passing because the user we return from the DAO already has SMS set as the `second_factor` method?

Reply:
Yeah, you're right that this is testing nothing as it stands.  I do think we should represent the situation with a (better) test though - what do you think the behaviour ought to be if we somehow receive an 'existing user' invite completion request with a (redundant) 2FA method in the body?

Reply:
I'd say as long as we're testing it works fine without a `second_factor` method provided we're fine, which I think we do in other tests. If we wanted to be really thorough we could check that the `second_factor` is ignored and not updated on the user but that doesn't strike me as particularly necessary.